tasks/install.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Add EPEL repository
+  package:
+    name: epel-release
+    state: latest
+  tags:
+    - skip_ansible_lint
+
+# Around 2017-07-12, installing epel-release stopped being idempotent. The
```

Comment:
I usually prefer the official packages for the reasons I mentioned there. In this case, it seems they have somewhat dropped the ball. In this case, it seems either way would be fine... any way you think will be easier to maintain/understand is fine. It might be helpful to open a bug report asking them to update it too https://bugs.centos.org

The advantage of using the official package still stands: everything is moving from a static mirrorlist to using metalink, which is a more dynamic service that reacts to the source IP and other parameters to give the best mirror list. We'd have missed that if we hardcoded the repo configuration... but now it recreates this other issue. Looks like we can't have our cake and eat it too :)